uvicorn/_handlers/http.py

```diff
@@ -7,6 +7,9 @@
     from uvicorn.server import ServerState
 
 
+MAX_RECV = 2 ** 16
```

Comment:
Is this value considered in the configuration?

Reply:
I don't see users have the need to customize the chunk size. The stream buffer limit in asyncio is `2 ** 16` 64 KiB [by default](https://github.com/python/cpython/blob/main/Lib/asyncio/streams.py#L22), `read` will never return a chunk larger than the buffer limit.